### PR TITLE
fix stylization of APY->APy, Fix #148

### LINKED ISSUE #147: Display APy endpoint being used in Sandbox

### LINKED ISSUE #148: APY vs APy stylization discrepancies
We should use `APy` since it was the original title and has more semantics to it.

### PR DESCRIPTION
**Pull request by share-with-me**
Fix  #148

### CODE REVIEW

File: assets/strings/eng.json [hunk 1]
```diff
@@ -39,9 +39,9 @@
     "Morphological_Generation_Help": "Choose a language and generate surface forms from your text using Apertium's morphological generators.",
     "Generate": "Generate",
     "Spell_Checker": "Spell checker",
-    "APY_Sandbox": "APY sandbox",
```
Comment (sushain97) [inline assets/strings/eng.json]: Corresponding changes need to be made in each `strings/*.json`

File: assets/strings/eng.json [hunk 2]
```diff
@@ -39,9 +39,9 @@
...
-    "APY_Sandbox_Help": "Send arbitrary requests",
-    "APY_Request": "APY Request",
+    "APy_Sandbox": "APy sandbox",
+    "APy_Sandbox_Help": "Send arbitrary requests to ",
```
Comment (sushain97) [inline assets/strings/eng.json]: This isn't amenable to localization.

File: assets/strings/eng.json (hunk 1 above)
Comment (share-with-me) [inline assets/strings/eng.json]: Oh yes.

File: assets/strings/eng.json (hunk 2 above)
Comment (share-with-me) [inline assets/strings/eng.json]: You mean the contents => `Send arbitrary request to `? Because of space?

Comment (sushain97) [inline assets/strings/eng.json]: You can't simply append the endpoint to this string in different languages. It won't alwas make sense.

Comment (share-with-me) [inline assets/strings/eng.json]: Um, how do you suggest I go about it? I initially thought of keeping a `:` or some symbol and then place the endpoint.

Comment (sushain97) [inline assets/strings/eng.json]: That approach doesn't work either for any language.

Comment (sushain97) [inline assets/strings/eng.json]: Probably need a different location for the endpoint

Comment (share-with-me) [inline assets/strings/eng.json]: Ohh. I'd experiment and paste snapshots here.

**Comment by share-with-me** [comment]
@sushain97 , I plan to open a separate PR for the display point. I would make this PR resolve #148 .

**Comment by sushain97** [comment]
Can you run this script and include the changes it makes: https://github.com/goavki/apertium-html-tools/blob/master/assets/strings/cleanup.sh

**Comment by share-with-me** [comment]
I ran the script and it outputted nothing on CLI. 

screenshot from 2017-08-18 09-27-38


> Can you run this script and include the changes it makes:

Um, include the changes where?

**Comment by sushain97** [comment]
It doesn't output anything to stdout. It makes changes to your working
directory.


[image: Sushain Cherivirala]
*Sushain K. Cherivirala *
Carnegie Mellon University, B.S. in Computer Science '17
Forward Deployed Software Engineer Intern, Palantir Technologies
Vice Chair, Undergraduate Student Senate
Research Assistant, Institute for Software Research
(713) 992-4043  <713-992-4043>| www.skc.name

**Comment by share-with-me** [comment]
I ran the script. I did a `git status` to see if it changed files. It did not show any files. How do I track the changes?

**Pull request merged**